src/main/webapp/app/shared/components/code-button/code-button.component.ts

```diff
@@ -120,7 +125,7 @@ export class CodeButtonComponent implements OnInit, OnChanges {
         if (this.useSsh && this.sshEnabled && this.sshTemplateUrl) {
             return this.getSshCloneUrl(this.getRepositoryUri()) || this.getRepositoryUri();
         }
-
+        console.log(this.versionControlAccessTokenRequired);
```

Comment:
Don't forget to remove logging later :) Or shall that stay?

Reply:
removed it :+1: